### ISSUE TITLE
Interface not showing up while watching a tv show

### ISSUE THREAD

**Issue by monheimx9** [CyferShepard/Jellystat#27]
Docker running on Windows 10
I'm fairly new to docker, I don't know if my installation went correctly

Interface die after refresh, even after restarting the container
Installed it few hours ago

Using that docker-compose file [https://github.com/CyferShepard/Jellystat/blob/main/docker-compose.yml](url)

Happened during while watching a tv show, I took a break mid watching, about 15min, then saw the issue showed bellow
Tested on Firefox, Chrome, Edge, same results
Stopped after the episode was over

A gif might help understanding what I mean
![Alt Text](https://i.imgur.com/nmWkSxw.gif)

__Anyway_ thank you, excited to see how it will evolve :^)_

**Comment by monheimx9** [comment]
After multiple attemps, I wasn't able to reproduce that behavior
Sadly I have a bad habit to clear caches/cookies/storage from the browser when it starts doing weird things, that's what I did yesterday

I'll try again and let you know if it happen again
Let's wait a few days and if nothing happen, maybe we can close that issue, might have been a one time thing
If I recall correctly I launched the "Synchronize with Jellyfin task" (for the first time) before it happened but can't be sure if it happend during, or right after

**Comment by CyferShepard** [comment]
Hi @monheimx9 , no worries, someone else is having a similar issue aswell, im currently looking into it so hopefully it doesn't come up again

**Comment by CyferShepard** [comment]
Possibly Caused by #18 , need to investigate

**Comment by MrCaringi** [comment]
I am faceing the same issue,
after login, the screen is blank, 
when I press F5 it shows the interface for a few milliseconds

I tried with firefox and chrome

**Comment by CyferShepard** [comment]
Hi @MrCaringi , will you be able to send me the logs? Press F12 in your browser, go to the console and send me a screenshot of whats there after you reload

**Comment by CyferShepard** [comment]
Can you also tell me if either Jellystat or Jellyfin is proxied with a baseurl. eg jellystat.local/jellystat/ or jeelyfin.local/jellyfin/ or anything similar

**Comment by MrCaringi** [comment]
> 

Sure!

![2023-05-27_160058](https://github.com/CyferShepard/Jellystat/assets/47096567/912e2b3f-488f-40cd-b449-8b8efdab666f)

thanks for your support!

**Comment by MrCaringi** [comment]
> Can you also tell me if either Jellystat or Jellyfin is proxied with a baseurl. eg jellystat.local/jellystat/ or jellyfin.local/jellyfin/ or anything similar

Im using it locally (local DNS in my LAN)

**Comment by CyferShepard** [comment]
Hmm, this looks like its trying to show an item that was recently added that does not have an image, can you let me know if this is the case? in the mean time il try to address that issue

**Comment by CyferShepard** [comment]
Hi @MrCaringi , a fix has been pushed, can you let me know if the latest version helps with this error?

**Comment by MrCaringi** [comment]
> Hi @MrCaringi , a fix has been pushed, can you let me know if the latest version helps with this error?

Sorry for late reply!

Yes! I had a movie without thumbnails (the folder was missing and there was a error when I tried to remove it).
I fix this issue (recreating the missing path and rescanning, then I was able to delete it successfully from web-interface)
but this does not fix the problem.

Then I pulled the lastest docker image and it worked!!!

thanks a lot!
